### PR TITLE
Add Dockerfile

### PR DESCRIPTION
**Pull request by hectorj2f**
ping @zeisss @marians @giantswarm/team-core

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,10 @@
 # Yochu
 
-[![Build Status](https://api.travis-ci.org/giantswarm/yochu.svg)](https://travis-ci.org/giantswarm/yochu) [![](https://godoc.org/github.com/giantswarm/yochu?status.svg)](http://godoc.org/github.com/giantswarm/yochu) [![IRC Channel](https://img.shields.io/badge/irc-%23giantswarm-blue.svg)](https://kiwiirc.com/client/irc.freenode.net/#giantswarm)
+[![Build Status](https://api.travis-ci.org/giantswarm/yochu.svg)](https://travis-ci.org/giantswarm/yochu) [![](https://godoc.org/github.com/giantswarm/yochu?status.svg)](http://godoc.org/github.com/giantswarm/yochu)
+[![](https://img.shields.io/docker/pulls/giantswarm/yochu.svg)](http://hub.docker.com/giantswarm/yochu)
+[![IRC Channel](https://img.shields.io/badge/irc-%23giantswarm-blue.svg)](https://kiwiirc.com/client/irc.freenode.net/#giantswarm)
 
-`yochu` provisions already running CoreOS hosts with Docker, `etcd`, `fleet` and `iptables`.
+`yochu` provisions already running CoreOS hosts with Docker, `etcd`, `fleet`,`rkt`, `kubectl` and `iptables`.
```
Comment (zeisss) [inline README.md]: those are not added in this branch?

**Comment by zeisss** [comment]
Can you tell me about the use-case for this?

**Comment by hectorj2f** [comment]
@zeisss why shouldn't someone be willing to run yochu as a container ? Of course, this limit its usage to setup the docker daemon, but what about the other steps: etcd, fleetd, rkt and k8s.? It might be a corner case thought

**Comment by zeisss** [comment]
oh, so running inside the container but modifying the host which would be passed in as a volume?

**Comment by hectorj2f** [comment]
@zeisss I'll close this PR, we might not have anyone interested on using this execution alternative at this moment. :)

**Comment by zeisss** [comment]
agreed ;)

**Pull request closed without merging**